SPECS/xen.spec

```diff
@@ -1210,6 +1256,11 @@ fi
 %endif
 
 %changelog
+* Tue May 21 2019 Benjamin Reis <benjamin.reis@vates.fr> - 4.7.5-5.9.1xcp
+- Security update
+- Fix XSA-297
```

Comment:
+ "Patches backported from xen in XCP-ng 7.6"

Reply:
Done